core/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -353,8 +396,11 @@ public OpenTelemetryRumBuilder setServiceManager(ServiceManager serviceManager)
      * If not specified, the default schedule exporter will be used.
      */
     public OpenTelemetryRumBuilder setExportScheduleHandler(
-            ExportScheduleHandler exportScheduleHandler) {
-        this.exportScheduleHandler = exportScheduleHandler;
+            @NonNull ExportScheduleHandler exportScheduleHandler) {
+        requireNonNull(exportScheduleHandler, "exportScheduleHandler cannot be null");
```

Comment:
I'm not sure we need both the annotation and the runtime check, do we?

Reply:
I find it best to be clear with your assertions, otherwise it might crash down the line. FWIW Kotlin implements non nullable parameters like this. I can remove if you prefer.

Reply:
I don't think it's a big deal, I just thought it was surprising to see both. Let's continue with it. 👍🏻